Widgets/2-FAQs.md

```diff
@@ -53,19 +53,21 @@ Yes, it is possible to make widgets visible on additional pages. The Sezzle widg
 Yes, the widget and modal have built-in translations in French, German, and Spanish for multilingual websites. By default, translation occurs based on the user's default browser language. Update the Language Settings in the Chrome Extension based on how the translation should occur. If the widget is translating but not the modal, please reach out to Sezzle Support. No other languages are supported at this time.
 
 ## Can the color of the Sezzle logo be changed?
-Yes, you can update the logo to coordinate and contrast with the background color of your website. The Sezzle logo `theme` options available are `dark`, `light`, `grayscale`, `black-flat`, `white`, `white-flat`, `purple-pill`, or `white-pill`. However, it is against our branding guidelines to otherwise customize the logo color.
+Yes, you can update the logo to coordinate and contrast with the background color of your website. The Sezzle logo `theme` options available are `dark`, `light`, `grayscale`, `black-flat`, `white`, `white-flat`, `purple-pill`, `pride-white-pill`,`pride-purple-pill`,`white-pill`. However, it is against our branding guidelines to otherwise customize the logo color.
```

Comment:
`pride-pill`, not `pride-purple-pill`.  

`pride-whitepill`, not `pride-white-pill`.

I'd also prefer if similar options stayed together:

`dark`, `light`, `grayscale`, `black-flat`, `white`, `white-flat`, `purple-pill`,`white-pill`, `pride-whitepill`, or `pride-purple-pill`